src/components/Wallet/WalletGenerator.jsx

```diff
@@ -265,17 +416,25 @@ function mapStateToProps(state) {
     ...state.quorum,
     ...state.wallet,
     ...state.wallet.common,
+    unknownAddresses: getUnknownAddresses(state),
+    unknownSlices: getUnknownAddressSlices(state),
```

Comment:
This is where we use the selectors. The selectors take the state and either return a memoized value or recompute the value we need from the state. Then putting this in the `mapStateToProps` passes the computed data from state to the component as a prop.